app/selectors/app.js

```diff
@@ -50,6 +50,12 @@ const selectMailTo = () =>
     appState => appState.get('mailTo')
   )
 
+const selectPrisonStaffHubUrl = () =>
```

Comment:
One for later but i've been trying to move away from selectors. I have been converting the matchPropsToSate to use the state directly.

Reply:
yeah they're not the best but changing the pattern wasn't a route I wanted to go down for this ticket.